src/pieces.py

```diff
@@ -9,118 +9,272 @@
     Rook: Moves along ranks and files. Worth 5.
     Queen: Moves both like a rook and bishop. Worth 9.
     King: Moves one square in any direction. Worth N/A (you loose the king it's game over).
+
+Variable annotations:
+    n: north
+    w: west
+    s: south
+    e: east
+
+Concatenation of aforementioned annotations means combination of directions (see self-documenting code).
+Trailing '_' means indefinite.
 """
 
 from dataclasses import dataclass
+from enum import Enum
+from itertools import product, cycle
```

Comment:
Unused imports.

Please check all your imports on all files and make sure to sort them based on PEP8 (right click "sort imports)

Reply:
Wait, but I do use `Enum` for `Color` and `product` for `steps` generation in some cases. `cycle` is indeed unused. I'll double-check.

Reply:
Per the ordering, as long as our list of imports is not gigantic, I prefer a more semantic order, I'll see what PEP8 order does anyway.

Reply:
`cycle` and `Iterable` are unused.